brave/common/extensions/url_bindings.cc

```diff
@@ -300,6 +304,13 @@ void URLBindings::New(const v8::FunctionCallbackInfo<v8::Value>& args) {
 
 void URLBindings::FormatForDisplay(
     const v8::FunctionCallbackInfo<v8::Value>& args) {
+  this->FormatForDisplay(args, net::UnescapeRule::NONE);
+}
+
+void URLBindings::FormatForDisplay(
```

Comment:
also, did you forget to add a file to this commit? `net::UnescapeRule` can't be used by JS without a gin converter